site/app/templates/officeHoursQueue/QueueAnnouncement.twig

```diff
@@ -15,7 +15,16 @@
       </a>
     </span>
   </label>
+  <br/>
+  <span id="markdown_buttons">
+    <button type="button" title="Insert a link" onclick="addMarkdownCode(1, $('#queue-announcement-message'))" class="btn btn-default btn-markdown key_to_click"  tabindex="0">Link <i class="fas fa-link fa-1x"></i></button>
+    <button title="Insert a code segment" type="button" onclick="addMarkdownCode(0, $('#queue-announcement-message'))" class="btn btn-default btn-markdown key_to_click"  tabindex="0">Code <i class="fas fa-code fa-1x"></i></button>
+    <button title="Insert bold text" type="button" onclick="addMarkdownCode(2, $('#queue-announcement-message'))" class="btn btn-default btn-markdown key_to_click"  tabindex="0">Bold <i class="fas fa-bold fa-1x"></i></button>
+    <button title="Insert italic text" type="button" onclick="addMarkdownCode(3, $('#queue-announcement-message'))" class="btn btn-default btn-markdown key_to_click"  tabindex="0">Italics <i class="fas fa-italic fa-1x"></i></button>
+    <button title="Preview Markdown" type="button" onclick="previewQueueMarkdown.call(this)" class="btn btn-default btn-markdown key_to_click" tabindex="0">Preview <i class="fas fa-eye fa-1x"></i></button>
+  </span>
```

Comment:
So this is another place where it might be good to consider how we might abstract these duplicated buttons into a twig template (or more ideally abstract the entire buttons + textarea + preview) so that any place we want to use this, we only need to edit one place instead of the 4 or so (right now).